org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.domainmodel/build.properties

```diff
@@ -2,10 +2,19 @@ source.. = src/,\
            src-gen/,\
            xtend-gen/
 bin.includes = model/generated/,\
-               model/,\
-               META-INF/,\
                .,\
+               META-INF/,\
                plugin.xml,\
-               about.html
-additional.bundles = org.eclipse.xtext.generator,\
-                     org.apache.commons.logging
+               about.html,\
+               model/
+additional.bundles = org.eclipse.xtext.xbase,\
+                     org.eclipse.xtext.common.types,\
+                     org.eclipse.xtext.xtext.generator,\
+                     org.eclipse.emf.codegen.ecore,\
+                     org.eclipse.emf.mwe.utils,\
+                     org.eclipse.emf.mwe2.launch,\
+                     org.eclipse.emf.mwe2.lib,\
+                     org.objectweb.asm,\
+                     org.apache.commons.logging,\
+                     org.apache.log4j,\
+                     com.ibm.icu
```

Comment:
Cannot recall how this came in. Reduced the build dependencies so that the workflow is executed without errors.